egov/egov-egi/src/main/java/org/egov/infra/workflow/entity/StateHistory.java

```diff
@@ -269,5 +273,31 @@ public Position getInitiatorPosition() {
     public void setInitiatorPosition(Position initiatorPosition) {
         this.initiatorPosition = initiatorPosition;
     }
+    
+    public Task map() {
+        Task t=new Task();
+        t.setBusinessKey(this.getState().getType());
+        t.setComments(this.comments);
+        t.setCreatedDate(this.getCreatedDate());
+        t.setId(this.getId().toString());
+        t.setStatus(this.getValue());
+        t.setDescription(this.getNatureOfTask());
+        t.setDetails(this.extraInfo==null?"hello":this.extraInfo);
+        t.setSender(this.senderName);
+        t.setOwner(this.getOwnerPosition().getId().toString());
+        return t;
+        
```

Comment:
t.setDetails(this.extraInfo==null?**"hello"**:this.extraInfo); hello ?
What is the purpose of this API in StateHistory and State ?